contracts/partial/vault/vault_lambdas.ligo

```diff
@@ -55,8 +54,8 @@ function deposit(
       }
```

Comment:
Decrease the TVL and virtual balance by params.amount. Otherwise,  TVL can sometimes be bigger than wrapped token supply